IsraelHiking.Web/sources/common.css

```diff
@@ -102,6 +107,12 @@ body {
     text-align: left !important;
 }
 
+/*#region margins*/
+.margin-right-10 {
+    margin-right: 10px;
+}
+/*#endregion*/
+
 @media print {
```

Comment:
Have you tested this in both Hebrew and English? I'm don't remember if margin-right works correctly in RTL and LTR.
I'll merge it in any case and write you comments in #349.